docs/_config.yml

```diff
@@ -0,0 +1,98 @@
+#     Plugins:
+#       jekyll-commonmark-ghpages
+#  
+#     commonmark:
+#       options: ["SMART", "FOOTNOTES"]
+#       extensions: ["strikethrough", "autolink", "table", "tagfilter"]
+#
+# However, it doesn't work well with Jekyll and the Just-The-Docs theme,
+# and causes too many formatting problems. Additionally, it also doesn't
+# have any support for auto-generating a Table of Contents (TOC) section
+# for a page. Due to these reasons, we instead use the recommended
+# markdown renderer for Jekyll, called kramdown.
+#
+markdown: kramdown
+
+# Notes on plugins:
+#  - GitHub Pages ignores all plugins included in the Gemfile.
+#    - Only the ones listed in the _config.yml file (this file) are used.
+#  - There is a very limited set of plugins that GH Pages supports (for security
+#    reasons). The listing is here: https://pages.github.com/versions
+#  - The jekyll-sitemap plugin generates a "sitemap.xml" for search engines.
+#  - We need the "jemoji" plugin for rendering emoji like ":point_right:".
+plugins:
+  - jekyll-remote-theme
+  - jekyll-sitemap
+  - jemoji
+
+search_enabled: true
+
+# Color scheme currently only supports "dark" or nil (light).
+color_scheme: nil
+
+# Aux links for the upper right navigation
+aux_links:
+  "API Docs":
+    - "//unicode-org.github.io/icu-docs"
+  "ICU on GitHub":
+    - "//github.com/unicode-org/icu"
+  "ICU Home Page":
+    - "http://site.icu-project.org/"
+  "Unicode Home Page":
+    - "//www.unicode.org"
+
+# Makes Aux links open in a new tab.
+aux_links_new_tab: true
+
+# Add a "Back to top" link at the bottom of each page.
+back_to_top: true
+back_to_top_text: "Back to top"
+
+# Enable heading anchors
+heading_anchors: true
+
+# Footer content: appears at the bottom of every page after the main content.
+footer_content: "Copyright &copy; 2016 and later Unicode, Inc. and others. All Rights Reserved. <a href=\"http://www.unicode.org/copyright.html\">Terms of use and License</a>."
+
+# Footer content: Add a "Edit this page on GitHub" link.
+# Note: I think this would be better at the *top* of the page, but it isn't possible to do that by default with just-the-docs.
+gh_edit_link: true # show or hide edit this page link
+gh_edit_link_text: "Edit this page on GitHub."
+gh_edit_repository: "https://github.com/unicode-org/icu"
+# NOTE: Since we serve the docs out of the "docs" folder, we need to include that here even though
+# it is technically *not* part of the branch name.
+gh_edit_branch: "master/docs"
+# Can be "tree" or "edit". Edit takes the user to jump into the editor immediately.
+gh_edit_view_mode: "tree"
+
+# Exclude from processing.
+# The following items will not be processed, by default.
+# Any item listed under the `exclude:` key here will be automatically added to
+# the internal "default list".
+# Excluded items can be processed by explicitly listing the directories or
+# their entries' file path in the `include:` list.
+exclude:
+  - Gemfile
+  - Gemfile.lock
```

Comment:
There are a lot more notes in our ICU User Guide Sites -> Markdown document about setup, what worked, and what didn't, which explains why we have some of the things we have in `_config.yml`.  I'm not sure where & how to put that "meta-documentation", but it would be nice for people who want to make changes to know where to find that info.

Reply:
Hmm, good point!
Do you think maybe adding a link to the [doc](https://docs.google.com/document/d/1uK91cxv1amCrt75TBw1PlCC5wZhJH_w3dW_8unWL9EQ/edit#bookmark=id.hefmm8pec6zd) in the `_config.yml` file comments would work maybe?

Reply:
Once you work on the "how to edit the user guide" page you could add a link from there to the migration doc for historical details etc.

Reply:
I like that idea -- now that I know that there is already a "how to edit the user guide", it makes sense as a natural place to link to the migration doc.  And it might be easier to land this PR first and create a followup PR for making that change. So I'll resolve this PR convo.

Reply:
Sounds good -- thank you both!